packages/amplify_datastore_plugin_interface/lib/src/publicTypes.dart

```diff
@@ -1,8 +1,20 @@
 export 'types/exception/DataStoreException.dart';
 export 'types/exception/DataStoreExceptionMessages.dart';
+export 'types/models/auth_rule.dart';
+export 'types/models/model.dart';
+export 'types/models/model_field.dart';
+export 'types/models/model_field_definition.dart';
+export 'types/models/model_field_type.dart';
+export 'types/models/model_provider.dart';
+export 'types/models/model_schema.dart';
+export 'types/models/model_schema_definition.dart';
+export 'types/models/subscription_event.dart';
+export 'types/models/uuid.dart';
+export 'types/query/query_field.dart';
 export 'types/sync/DataStoreSyncExpression.dart';
+export 'types/temporal/datetime_parse.dart';
 export 'types/temporal/temporal_date.dart';
 export 'types/temporal/temporal_time.dart';
 export 'types/temporal/temporal_datetime.dart';
 export 'types/temporal/temporal_timestamp.dart';
-export 'types/query/query_field.dart';
+export 'types/utils/parsers.dart';
```

Comment:
I was looking into this file working on something else, there is one util function used only by internal functions. Not sure if should export it at all...

Reply:
Ok, will remove this one for sure. The more I look at this look, I'm not sure if anything besides `SubscriptionEvent` really needs to be added, but will try to cull this list a little.